Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java

```diff
@@ -63,7 +63,7 @@ public class VerifyCardDataTest {
 
     private static final Logger logger = Logger.getLogger(VerifyCardDataTest.class);
 
-    private static final String FULL_ABILITIES_CHECK_SET_CODES = "WOE"; // check ability text due mtgjson, can use multiple sets like MAT;CMD or * for all
+    private static final String FULL_ABILITIES_CHECK_SET_CODES = "*"; // check ability text due mtgjson, can use multiple sets like MAT;CMD or * for all
```

Comment:
The reason to not use * here in master is it'll make the logs unwieldy. Someday, once we address a bunch more text discrepancies.